BEP1.md

```diff
@@ -47,12 +56,34 @@ Other exceptional statuses include:
 - **Stagnant**: A BEP has not been updated for more than 6 months, it will enter Stagnant state.
 - **Withdrawn**: A BEP that is dropped and will not be implemented. Usually, it is due to some prerequisite conditions that are not true anymore.
 
-## 5.  Reference
+## 5.  BEP Format
+It is important to keep BEP clear and well organized, BEPs need to follow this format(BEP1 is excepted):
+- Preamble: a short metadata about the BEP, it should be put at the top of the BEP. Here is an example:
+<pre>
+  BEP: 127
+  Title: Temporary Maintenance Mode for Validators
+  Status: Enabled
+  Type: Standards
+  Created: 2022-01-10
+  Author(optional): it can be a name or an email.
+  Description(optional): enabled by BSC Euler upgrade, block height 18907621, Jun-22-2022
```

Comment:
It is a bit wired to write when it is enabled in a `Description`

Reply:
maybe `Description` be renamed to `Additional Info`?